src/test/resources/com/atomist/rug/test/gherkin/handler/event/EventHandlersAgainstGenerated.ts

```diff
@@ -26,3 +29,19 @@ class ReturnsEmptyPlanEventHandlerGen1 implements HandleEvent<GraphNode, Build>
     }
 }
 export const handler1 = new ReturnsEmptyPlanEventHandlerGen1();
+
+@EventHandler("ReturnsEmptyPlanEventHandlerGenWithArrays", "Handles a new Commit event",
+    new PathExpression<GraphNode, Push>(
+        `/Push()`))
+@Tags("push")
+class ReturnsEmptyPlanEventHandlerGenWithArrays implements HandleEvent<GraphNode, Push> {
+
+    handle(m: Match<GraphNode, Push>) {
+        let p: Push = m.matches()[0]
+
+        if (p.contains().length != 1) throw new Error("No contains")
```

Comment:
That a boolean-like method does return an array looks odd, from a developer perspective, don't you think?